tests/templates/test_tensornetworks/test_MPS.py

```diff
@@ -14,120 +14,284 @@
 r"""
 Tests for the MPS template.
 """
+import math
 import pytest
-import random
 import numpy as np
 import pennylane as qml
 from pennylane.templates.tensornetworks.mps import *
 
-"""
-Unit tests for the MPS template.
-"""
 
-class TestIndicesMPS: 
+class TestIndicesMPS:
     """Test function that computes MPS indices"""
-    
-    def test_exception_wrong_dim(self):
-        """Verifies that exception is raised if the number of wires is incorrect."""
-        
-        n_wires = random.randrange(1,20,2)
-        loc = 2
-        with pytest.raises(AssertionError, match="number of wires must be an even integer; got {}".format(n_wires)):
-            compute_indices_MPS(range(n_wires), loc)
 
-        n_wires = random.randrange(2,20,2)
-        loc = random.randrange(1,n_wires,2)
-        with pytest.raises(AssertionError, match="loc must be an even integer; got {}".format(loc)):
-            compute_indices_MPS(range(n_wires), loc)
+    @pytest.mark.parametrize(
+        ("n_wires", "n_block_wires"),
+        [
+            (5, 3),
+            (9, 5),
+            (11, 7),
+        ],
+    )
+    def test_exception_n_block_wires_uneven(self, n_wires, n_block_wires):
+        """Verifies that an exception is raised if n_block_wires is not even."""
+
+        with pytest.raises(
+            ValueError, match=f"n_block_wires must be an even integer; got {n_block_wires}"
+        ):
+            compute_indices_MPS(range(n_wires), n_block_wires)
+
+    @pytest.mark.parametrize(
+        ("n_wires", "n_block_wires"),
+        [
+            (3, 4),
+            (6, 8),
+            (10, 14),
+        ],
+    )
+    def test_exception_n_block_wires_large(self, n_wires, n_block_wires):
+        """Verifies that an exception is raised when n_block_wires is too large."""
+
+        with pytest.raises(
+            ValueError,
+            match=f"n_block_wires must be smaller than or equal to the number of wires; got n_block_wires = {n_block_wires} and number of wires = {n_wires}",
```

Comment:
line is too long. Should maybe run `black` on this file

Reply:
Solved in commit https://github.com/PennyLaneAI/pennylane/pull/1871/commits/d6c7dc41bbaff2f5560af3947571640978a7f35c